Packs/CommonTypes/ReleaseNotes/3_3_23.md

```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Fields
+- **Traffic Light Protocol**
```

Comment:
Please add a description of the change - for example, the added values

Reply:
Done